### PR TITLE
documentation/basic-ad-update

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
## PR Overview
**This PR will address the following Issue/Feature:** #26 and [#35](https://github.com/fivetran/dbt_facebook_ads/issues/35)

**This PR will result in the following new package version:** `v0.7.1`

This will not be a breaking change as this only updates documentation.

**Please detail what change(s) this PR introduces and any additional information that should be known during the review of this PR:**

### Documentation Updates
- The prerequisite steps for generating the `basic_ad` pre-built report have been modified to reflect the current state of the Facebook Ads connector.
- Adds the [DECISIONLOG](DECISIONLOG.md) to clarify why there exist differences among aggregations across different grains.

### Additional updates
- This PR includes the addition of the auto releaser and 2nd reviewer github actions to be leveraged following merge to `main`.

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [n/a] dbt compile
- [n/a] dbt run –full-refresh
- [n/a] dbt run
- [n/a] dbt test
- [n/a] dbt run –vars (if applicable)

Before marking this PR as "ready for review" the following have been applied:
- [X] The appropriate issue has been linked and tagged
- [X] You are assigned to the corresponding issue and this PR
- [X] BuildKite integration tests are passing

### Detailed Validation
Please acknowledge that the following validation checks have been performed prior to marking this PR as "ready for review":
- [X] You have validated these changes and assure this PR will address the respective Issue/Feature.
- [X] You are reasonably confident these changes will not impact any other components of this package or any dependent packages.
- [X] You have provided details below around the validation steps performed to gain confidence in these changes.

In order to validate these steps I went through the creation of a new Facebook Ads connector `facebook_ads_basic_ad_test` and confirmed the `basic_ad` pre-built report is now synced by default and if you wish to enable/disable the table you will do so via the schema tab.

### Standard Updates
Please acknowledge that your PR contains the following standard updates:
- Package versioning has been appropriately indexed in the following locations:
    - [X] indexed within dbt_project.yml
    - [X] indexed within integration_tests/dbt_project.yml
- [X] CHANGELOG has individual entries for each respective change in this PR
    
- [X] README updates have been applied (if applicable)
    
- [X] DECISIONLOG updates have been updated (if applicable)
- [n/a] Appropriate yml documentation has been added (if applicable)

### dbt Docs
Please acknowledge that after the above were all completed the below were applied to your branch:
- [n/a as no code or yml changes were applied] docs were regenerated (unless this PR does not include any code or yml updates)

### If you had to summarize this PR in an emoji, which would it be?
 
🏗️

### CODE REVIEW

**Review by fivetran-catfritz — APPROVED** [review]
lgtm!

**Review by fivetran-catfritz — APPROVED** [review]

**Review by fivetran-catfritz — APPROVED** [review]

**Comment by fivetran-catfritz** [comment]
@fivetran-joemarkiewicz Hey really really really sorry I accidentally committed to your branch... I undid it though so you can ignore.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+# dbt_facebook_ads_source v0.7.1
+
+[PR #31](https://github.com/fivetran/dbt_facebook_ads_source/pull/31) includes the following updates:
+## Documentation Updates
+- The prerequisite steps for generating the `basic_ad` pre-built report have been modified to reflect the current state of the Facebook Ads connector.
```
Comment (fivetran-reneeli) [inline CHANGELOG.md:5]: May be worth specifying where the steps are (so linking the README), and provide more context on what the current state is, so people can just look at the CHANGELOG to understand without having to click further

Comment (fivetran-joemarkiewicz) [reply CHANGELOG.md]: Good call. Just updated.

**Review by fivetran-reneeli — APPROVED** [review]
lgtm!

**Pull request merged**